test/ipc_os_prov_consumer.c

```diff
@@ -25,9 +25,6 @@ int main(int argc, char *argv[]) {
 
     os_params = umfOsMemoryProviderParamsDefault();
     os_params.visibility = UMF_MEM_MAP_SHARED;
-    if (argc >= 3) {
-        os_params.shm_name = argv[2];
-    }
```

Comment:
IMHO it should stay as it was. The consumer can use also another SHM file or none.